src/t2iapi/combined/service.proto

```diff
@@ -62,4 +62,14 @@ service CombinedService {
   rpc SetModeOfOperationAndSetOperatingMode (
   t2iapi.combined.SetModeOfOperationAndSetOperatingModeRequest)
       returns (BasicResponse);
+
+  /*
+  Request to set the activation state of the system context with the specified handle to the requested value and to set
```

Comment:
I would prefer wording @ActivationState and @ContextAssociation as mostly used in this file.

Reply:
changed